app/src/androidTest/java/com/github/houseorganizer/houseorganizer/CreateHouseholdActivityTest.java

```diff
@@ -69,19 +81,64 @@ public void setupHouseholds() throws ExecutionException, InterruptedException {
 
         Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
         context.sendBroadcast(new Intent(Intent.ACTION_CLOSE_SYSTEM_DIALOGS));
+        createHouseholdRule.getScenario().onActivity(a -> decorView = a.getWindow().getDecorView());
     }
 
     @Test
     public void createHouseholdWorks() throws ExecutionException, InterruptedException {
+
+        final String houseName = "MyHouse";
```

Comment:
Create a variable at the start of the file such that you can reuse it

Reply:
Yes good idea correcting it !